api_calls_string.py

```diff
@@ -0,0 +1,455 @@
+import http.client
+import json
+import datetime
+import os.path
+import csv
+
+
+
+
+api_key = 'ry27AHhE2O4WbIO24c0DE4Zt1KeMvyMx5f31taX1'
+connection = http.client.HTTPSConnection("api.sportradar.us")

```

Comment:
I would remove either line 11 or 12 as there is redundancy as it’s the same connection but different var names